src/symbol/quads.js

```diff
@@ -99,93 +99,100 @@ export function getGlyphQuads(anchor: Anchor,
                        layer: SymbolStyleLayer,
                        alongLine: boolean,
                        feature: Feature,
-                       positions: {[string]: {[number]: GlyphPosition}},
+                       positions: {[string]: GlyphPositionData},
                        allowVerticalPlacement: boolean): Array<SymbolQuad> {
 
     const textRotate = layer.layout.get('text-rotate').evaluate(feature, {}) * Math.PI / 180;
 
     const positionedGlyphs = shaping.positionedGlyphs;
     const quads = [];
 
-    for (let k = 0; k < positionedGlyphs.length; k++) {
-        const positionedGlyph = positionedGlyphs[k];
-        const glyphPositions = positions[positionedGlyph.fontStack];
-        const glyph = glyphPositions && glyphPositions[positionedGlyph.glyph];
-        if (!glyph) continue;
+    if (shaping.lineCount === 0) return quads;
+    const lineHeight = (Math.abs(shaping.top) + Math.abs(shaping.bottom)) / shaping.lineCount;
```

Comment:
I think this might not work correctly in the case that `shaping.top` and `shaping.bottom` have the same sign. I'm not sure when that might be but maybe this should be `(bottom - top) / lineCount`?